Resources/Private/Javascript/Components/AddToCart.js

```diff
@@ -3,19 +3,9 @@ import { useApiClient } from '../Api/Context';
 const addToCartOnClick = ({ proxy, productId }) => {
   const apiClient = useApiClient();
 
-  proxy.addEventListener('click', async (event) => {
+  proxy.addEventListener('click', (event) => {
     event.preventDefault();
-    const result = await apiClient.addProductToCart(productId);
-
-    if(result.status == 200) {
-      document.dispatchEvent(new Event('cart-changed'));
-      document.dispatchEvent(new CustomEvent('added-to-cart', {
-        detail: {
-          element: proxy,
-          message: result
-        }
-      }));
-    }
+    apiClient.addProductToCart(productId);
```

Comment:
Is it really just that? I still think it is pretty useful to add a different state to the click-target, to prevent multiple clicks and give feedback to the user that something is happening

Reply:
I've added an utility class while loading